tests/system.rs

```diff
@@ -157,9 +156,11 @@ fn get_config(config_file: Option<&str>) -> Box<Config> {
         }
     };
 
-    let mut def_config_file = fs::File::open(config_file_name).ok().expect("Couldn't open config.");
+    let mut def_config_file = fs::File::open(config_file_name)
+                                  .ok()
+                                  .expect("Couldn\'t open config.");
     let mut def_config = String::new();
-    def_config_file.read_to_string(&mut def_config).ok().expect("Couldn't read config.");
+    def_config_file.read_to_string(&mut def_config).ok().expect("Couldn\'t read config.");
```

Comment:
Agreed, but it's not a direct result of this PR. The string is being reformatted where it wasn't earlier.